recipes/rpdb/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "rpdb" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5d1a1cee34378ab075879dc30fa6328d448a9f680a66c4e84cac7382ad92f15f
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - rpdb
+
+about:
+  home: http://tamentis.com/projects/rpdb
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: LICENSE.txt
```

Comment:
This needs to have the proper value to pick up the license file, see https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
I built this package locally and it worked fine. I've also verified that it does in fact include the LICENSE.txt file (and if you change the recipe to look for e.g. `LICENSE.foo` it fails as expected).

This is using conda-build version 3.17.6. If the long form is needed for historical reasons or compatibility with older versions, let me know and I'll make the change.

Reply:
We may in the futrure have to drop back to older versions of conda-build. Please feel free to open a PR to the docs to open up a discussion on the format of this line

Reply:
Went ahead and made the change. Seperately, opened https://github.com/conda-forge/conda-forge.github.io/pull/693 to discuss updating documentation.

Reply:
Thanks!